frontend/custom_version_strategy.rst

```diff
@@ -149,7 +148,7 @@ After creating the strategy PHP class, register it as a Symfony service.
                     '%kernel.project_dir%/busters.json',
                     '%%s?version=%%s',
                 ]
-        )->setPublic(false);
```

Comment:
PHP-DSL needed here

Reply:
What do you call php-dsl ? The usage of ContainerConfigurator ?

Reply:
Yes :)